src/Components/DoctorPage/Container/PatientCard/patientCard.less

```diff
@@ -0,0 +1,151 @@
+@media all {
+
```

Comment:
Are you using both styled-components and less? Please choose one approach

Reply:
it will be delete after create all styled component